Makefile

```diff
@@ -0,0 +1,112 @@
+BASE_REPO=docker.m8s.io/medallia
+AURORA_VERSION=0.13.0
+AURORA_RELEASE=$(AURORA_VERSION)-medallia-2
+# can be used to customize the release name (e.g. pre-release)
+GITHUB_AURORA_RELEASE=$(AURORA_RELEASE)
+AURORA_SNAPSHOT=https://github.com/medallia/aurora/archive/rel/$(AURORA_RELEASE).tar.gz
+AURORA_PACKAGE_BRANCH=0.13.x
+
+define fetch_aurora
+	cd scratch && \
```

Comment:
pushd/popd? I don't think the && is needed, IIRC make should fail if something returns !0